### DISCUSSION TITLE
Dub Anime

### DISCUSSION THREAD

**Discussion by CoderX-1** [ShivaBhattacharjee/AnimeTrix-next#9] [General]
Can we watch Dub Anime?

**Comment by ShivaBhattacharjee** [comment]
I am currently working on it, but I am busy with midterms and other commitments, so please expect a bit of delay.

**Reply by CoderX-1** [reply]
you can add checks if anime stream link =null dont display it

**Reply by CoderX-1** [reply]
i suggest you to check watch page
about dub

**Reply by ShivaBhattacharjee** [reply]
> i suggest you to check watch page
> about dub

Whats the issue?

**Reply by CoderX-1** [reply]
![image](https://github.com/ShivaBhattacharjee/AnimeTrix-next/assets/136450776/a98b067b-5c2a-4ce9-9e91-42e291e65f8d)
when i open dub ep when i have to change ep i have to check on db btn to change it to dub ep otherwise sub will open

**Comment by CoderX-1** [comment]
yesterday when i open your website in my old phone website theme got change into light i faced bad experience in that mobile no option to change system theme

**Reply by ShivaBhattacharjee** [reply]
See currently i am using tailwind classes to theme website based on device theme as the website is in experimental/ development stage some features might not work as expected. Taking your feedback  into consideration i will try to bring an option to force dark/light mode button

**Comment by ShivaBhattacharjee** [comment]
> Can we watch Dub Anime?

I have added dub anime:)

**Reply by CoderX-1** [reply]
you can add aniscan icon to search modal its look good

**Reply by johnx234** [reply]
i have ideas that like one anime have 3 season so you can link all season so user can access from any season they are watching it make easier for lazy user like me

**Reply by ShivaBhattacharjee** [reply]
> i have ideas that like one anime have 3 season so you can link all season so user can access from any season they are watching it make easier for lazy user like me
> 

The feature is already present "RELATIONS & SEASONS" lists manga as well as ova ona and other seasons too

**Reply by ShivaBhattacharjee** [reply]
> you can add aniscan icon to search modal its look good
> 
Thanks for the suggestion will implement it

**Reply by CoderX-1** [reply]
when i play dub anime the ep list showing me sub i have to click dub btn everytime while changing ep and below description audio is sub

**Reply by CoderX-1** [reply]
bro u can use disqus plugin for comment
